homeassistant/components/zwave_js/update.py

```diff
@@ -148,11 +164,11 @@ async def async_install(
         except BaseZwaveJSServerError as err:
             raise HomeAssistantError(err) from err
         else:
-            self._attr_installed_version = firmware.version
+            self._attr_installed_version = self._attr_latest_version = firmware.version
             self._latest_version_firmware = None
-            self._async_process_available_updates()
         finally:
             self._attr_in_progress = False
```

Comment:
This progress indicator is not better than the update integration built in indicator.

Reply:
This can be done in a follow up too.